package.json

```diff
@@ -150,4 +150,4 @@
     "vite": "^5.1.3",
     "vitest": "^1.3.1"
   }
-}
+}
```

Comment:
Can you revert this? And maybe check why this whitespace is removed by your editor.